### PR TITLE
Use of angular2-cookie

### PR DESCRIPTION
**Pull request by flaviencathala**

### CODE REVIEW

File: generators/client-2/templates/gulp/_copy.js [hunk 1]
```diff
@@ -113,6 +113,7 @@ function deps(){
         'node_modules/ui-router-ng1-to-ng2/**/*.js',
         'node_modules/ui-router-ng2/**/*.js',
         'node_modules/ui-router-visualizer/**/*.js',
+        'node_modules/angular2-cookie/**',
```
Comment (wmarques) [inline generators/client-2/templates/gulp/_copy.js]: we just need js files I think, so add /*.js

File: generators/client-2/templates/src/main/webapp/_system.config.js [hunk 2]
```diff
@@ -29,13 +30,14 @@
         'layouts' : 'app/layouts',
         'shared' : 'app/shared'
     };
-    // packages tells the System loader how to load when no filename and/or no extension
+        // packages tells the System loader how to load when no filename and/or no extension
```
Comment (wmarques) [inline generators/client-2/templates/src/main/webapp/_system.config.js]: fix indent

**Comment by flaviencathala** [comment]
@wmarques Done !

File: generators/client-2/templates/src/main/webapp/app/_app.ng2module.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { UIRouterModule, RootModule } from 'ui-router-ng2';
 import { Ng2Webstorage } from 'ng2-webstorage';
+import { CookieService } from 'angular2-cookie/services/cookies.service';
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/_app.ng2module.ts]: Why is this import required here?

**Comment by flaviencathala** [comment]
@deepu105 Sorry I made a mistake

**Comment by sendilkumarn** [comment]
LGTM 👍

File: generators/client-2/templates/src/main/webapp/app/account/_account.ng2module.ts [hunk 4]
```diff
@@ -80,6 +81,7 @@ let ACCOUNT_STATES = [
         SettingsComponent
     ],
     providers: [
+        CookieService,
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/account/_account.ng2module.ts]: Since it's declared in shared module already it's not required here.  Can you remove it with another pr
